syntax = "proto3";
package os.logger;

import "Common.proto";

message ControlOutput {
  Time  t = 1;
  uint32 frame_id  = 2;
  
  bool  target_too_close = 10;
  
  float x_error      = 20;
  float y_error      = 21;
  float yaw_error = 22;
  float yaw_adjusted_for_playback = 23;
  float yaw_target = 24;

  float vx_adjust        = 30;
  float vy_adjust        = 31;
  float vz_adjust        = 32;
  float yaw_adjust       = 33;
}
